types/react-typing-effect/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-typing-effect 2.0.4
```

Comment:
@rossdakin : any specific reason we have removed the patch version. I can see the source version is 2.0.4

"react-typing-effect": "^2.0.4",

Reply:
@Debananda – I added the `.4` but apparently I shouldn't have, according to [this rule](https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md) which caused the build to fail, so I reverted it to `2.0` as it was.